Projects/Foundations/Schemas/Docs-Nodes/C/Crypto/Crypto-Exchange/crypto-exchange.json

```diff
@@ -210,12 +291,12 @@
                     "text": "sandBox bazı borsalarda, tüccarların simüle edilmiş bir ortamda tüm borsa özelliklerini denemelerini sağlayan testnet/sandBox modu vardır. Etkinleştirmek için true olarak ayarlayın. Lütfen bazı borsaların (örneğin Bittrex gibi) bu parametrenin false olarak ayarlanmasıyla hata verebileceğini unutmayın. Bu gibi durumlarda o satırı silmeniz yeterlidir.",
                     "updated": 1652799269237
                 }
-            ],
-            "updated": 1650635009708
+            ]
```

Comment:
Did you make those deletions by hand or did they happen by accident ?

Reply:
Actually, this is not a deletion. It was in the wrong place in the file structure and it was moved to the right place. Line 282 in the file I sent is where it should be. It was moved there.

https://medium.com/superalgos/superalgos-translator-c76c6457b087
Translation work was done with this tool. I checked sentence by sentence and the Translator tool also corrects structural defects.

Reply:
Good that you sent me the article, I had not noticed anything about it.